assets/scripts/fe/src/app/commons/forms/add-password.forms.ts

```diff
@@ -0,0 +1,31 @@
+import { FormBuilder, FormGroup, FormControl, Validators } from '@angular/forms';
+
+
+export class AddPasswordForm {
```

Comment:
merge this class together with the EditPasswordForm class in a single file. you can name it `password.forms.ts`